embed-video.md

```diff
@@ -19,3 +19,58 @@
 - `muted`     : Sets the initial muted state. `true` or `false`. Defaults to `false`.
 - `autoplay`  : Automatically starts playing without the user clicking play. `true` or `false`. Defaults to `true`.
 - `time`      : Start playback at the given timestamp for videos ONLY. Must be in the format `1h2m3s` specifying hours, minutes, and seconds respectively. Defaults to the start of the video.
+
+###Video and Channel Metadata
+For video and channel metadata refer to the [Twitch API (Kraken)] (https://github.com/justintv/Twitch-API).
+For convenience, some available metadata is listed below for convenience. All are available by [`GET/videos/:id`] (https://github.com/justintv/Twitch-API/blob/master/v3_resources/videos.md#get-videosid). 
```

Comment:
Please do not duplicate information, it makes updating it a lot harder. Just link to the relevant documentation.